### ISSUE TITLE
ADD_FEEDING broke on v2.6.4

### ISSUE THREAD

**Issue by CajuCLC** [jcgoette/baby_buddy_homeassistant#90]
Hello there. I noticed that add feeding broke since using v2.6.4.
I have this code to add feeding based on the timer running:
```
      - show_name: true
        show_icon: true
        type: button
        tap_action:
          action: call-service
          service: babybuddy.add_feeding
          data:
            timer: true
            type: Formula
            method: Bottle
            amount: 6
          target:
            entity_id: switch.baby_timer
        entity: sensor.babybuddy_feedings
        name: Formula 6oz
        icon: mdi:baby-bottle
```
This is what it looks like on Home Assistant:
![image](https://user-images.githubusercontent.com/1249095/211190859-9d6de836-9e88-4eef-8f92-7d85716116dc.png)

Once I click Formula 6oz (or any other) I get this error:

```
Failed to call service babybuddy/add_feeding. BabyBuddyChildTimerSwitch.async_add_feeding() got an unexpected keyword argument 'type'
```

Looks like the issue is `async def async_add_feeding` on line 198 of the `switch.py`. And when we check line 200 of that file, there was a change of `type` to `feed_type`. Same thing on line 217.

**Comment by CajuCLC** [comment]
Actually, looking at it I see that add_diaper is also broken. `type` has been changed to `diaper_type`. Taking a look at other API calls, some more might be broken.

**Comment by jcgoette** [comment]
Thanks for this. Fixed in 6f22b469fc4e204bbcb17fc1abd9870d86629be1.
